docs/domain/id/index.md

```diff
@@ -87,6 +91,21 @@ public readonly partial record struct VeryLongEmployeeId;
 // The `VeryLongEmployeeId` will have format `employee_(guid)`, with `New` using `Guid.NewGuid` as random source.
 ```
 
+```cs
+[TypedId(TypedIdFormat.RawString, MaxValueLength = 100)]
+public readonly partial record struct ExternalId;
+
+// The `ExternalId` wraps any string up to 100 characters. Exposes `MaxLength` static property.
```

Comment:
Shouldn't we expose `MaxValueLength` to make it consistent with the prefixed variant?

Reply:
Not sure about this, the `Value` part of the name doesn't make much sense in this context, and consistency between types of ids doesn't really give us much. It is consistent with the attribute property name though. TBH I could see us going with both ways, and I've already backed down from the name you're proposing, but I can bring it back, if you think it makes more sense.